stripper/ze_icecap_escape_v5_p.cfg

```diff
@@ -1,3 +1,26 @@
+;Add a tonemap since apparently it can carry over between maps (and cause snow blindness)
+add:
+{
+	"classname" "env_tonemap_controller"
+	"targetname" "tonemap"
+	"origin" "-891.855 957.757 9"
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "logic_auto"
+	}
+	insert:
+	{
+		"OnMapSpawn" "tonemap,SetTonemapRate,-1,0,-1"
+		"OnMapSpawn" "tonemap,SetBloomScale,0,0,-1"
+		"OnMapSpawn" "tonemap,SetAutoExposureMin,0,0,-1"
+		"OnMapSpawn" "tonemap,SetAutoExposureMax,0,0,-1"
```

Comment:
These values were just grabbed from a css ported map that attempt to recreate a LDR look in a HDR compiled map (SetTonemapRate -1 disables autoexposure when moving between areas that are different brightnesses, SetBloomScale 0 disables bloom (which is mainly annoying around bright textures), and SetAutoExposureMin and SetAutoExposureMax force the autoexposure to a single value (since it is disabled with SetTonemapRate)). However, I can change it to use default HDR settings instead.